lib/avalanche.ex

```diff
@@ -55,22 +55,39 @@ defmodule Avalanche do
                               required: true,
                               doc: "Snowflake authentication via OAuth token or Key Pair."
                             ],
-                            poll_options: [
+                            poll: [
```

Comment:
why the rename? when you had `options` in the code below the intent was clearer imo. Maybe you want to move poll to a `%Poll{}` struct and this is the first step?

Reply:
Since these can be overridden on a single call it just makes the args verbose. I was trying to simplify the interface.

```
Avalanche.run("select ?;", [3], [request_id: request_id, retry: true], [poll: [max_attempts: 100], get_partitions: [timeout: 200_000])
```

vs.

```
Avalanche.run("select ?;", [3], [request_id: request_id, retry: true], [poll_options: [max_attempts: 100], get_partitions_options: [timeout: 200_000])
```